src/components/Button/Button.scss

```diff
@@ -487,6 +521,10 @@ $partial-button-filled-pressed-box-shadow: inset 0 0 0 0 transparent,
     padding-right: spacing(base-tight);
   }
 
+  &.disclosure {
```

Comment:
Noticed while replacing web that when there's iconOnly and Disclosure the spacing is off. Can be seen in the toolbar. Screenshot coming.